data/human/human missions.txt

```diff
@@ -4175,3 +4175,322 @@ mission "Hai Wormhole Warning"
+			choice
+				`	"I'd love to visit your family."`
+					goto parents
+				`	"I'm leaving again soon, but you're welcome to join me."`
+					goto companion
+				`	"I think I'm too busy for either of those right now."`
+					goto goodbye
+			label parents
+			`	Charlie pays for your train ticket to the countryside. In less than an hour, you arrive at his family's home. The farmhouse doesn't look old, but its red paint is weathered. "We don't own any of the surrounding land anymore," Charlie says, "But my parents kept the house with their jobs at the factory. Once I get my stuff paid for, I can help them out."`
+			`	His family greets you in the front yard. Posturing gruffly with his suspenders, his dad thanks you for sending Charlie home; fussying over his untidy hair, his mom insists you stay for dinner. Charlie's younger sister clings to her dad's leg until she gives you a flower.`
+			`	After a simple dinner, Charlie approaches you. "One last thing. I was being serious before. It'd be fun to travel the galaxy with you, if you're interested. We could do some good work together. How about it?"`
+			choice
+				`	"I'd love to have you as a member of my fleet."`
+					goto yep
+				`	"I think you should carve your own story, Charlie. I wouldn't want to hold you back."`
+					goto nah
+			label yep
+			`	"Heck yeah! I'll get my things!" You wait as he rushes around his home, quickly throwing together a suitcase of hopefully clean laundry. His mom makes you promise you'll take care of her son, especially against pirates. Soon, you and Charlie the train back to the spaceport together."`
+				accept
+			label nah
+			`	"Oh, stop it," he blushes. "Well, it's been fun. I hope your adventures are long and fun. You deserve it."`
+				decline
+			label companion
+			`	"Alright, I'll get my things ready!"`
+				accept
+			label goodbye
+			`	"Alright. Anyway, it's been real. See you!" Charlie jogs back across the spaceport and trips spectacularly.`
+				decline
+	on complete
+		conversation
+			`	"Hey, <first>," Charlie says. "I really enjoyed traveling with you, but I think I should check back in with my family and maybe fly solo for a bit. Anyway, I'll see you around!"`
+	on fail
+		conversation 
+			`	Your comms crackle as Charlie's voice calls, "I'm hit! I'm hit! I..."`
+			`	The feed cuts out.`
+
+mission "Kindred Spirit 5"
+	name "Charlie Companion"
+	description "Travel with Charlie for as long as you want."
+	repeat	
+	source "Heartland"
+	to offer
+		has "Kindred Spirit 4: done"
+		random < 50
+	npc save
+		government "Merchant"
+		personality heroic escort
+		ship "Quicksilver" "Tyran the Brave"
+	on offer
+		conversation
+			`	"Hey, <first>!" Charlie calls. "You want to travel together again?"`
```

Comment:
This should probably have a little more to it. Something about you wandering through the spaceport or similar when your comm buzzes or beeps or something with a call from Charlie. As it is, it sounds like he calls immediately as we enter the spaceport, which is kinda creepy.
Or, perhaps give us the option to contact him.
`You remember this is the planet where Charlie, a young pilot you've escorted in the past and who previously joined you on your travels lives. Contact him and see if he is interested in travelling together again?`